docs/users/bespoke-executor.md

```diff
@@ -153,10 +153,10 @@ explicitly or implicitly.
 Molecules may also be submitted from a chemical structure file:
 
 ```sh
-openff-bespoke executor submit --input molecule.sdf
+openff-bespoke executor submit --file molecule.sdf
```

Comment:
I'll see what I can do about adding tests for shell commands to the testing framework